exec/src/klio_exec/cli.py

```diff
@@ -39,7 +39,8 @@
 
 
 RuntimeConfig = collections.namedtuple(
-    "RuntimeConfig", ["image_tag", "direct_runner", "update", "blocking"]
+    "RuntimeConfig",
```

Comment:
I don't think any of the changes to the `klio-exec` code made in this PR are needed; it's taken care of in #197 . As well, we should not be telling the klio-exec to actually use the DirectGKERunner. By the time we are in this code path, we're already in the container on GKE. So we should be running using the plain direct runner.